pos_partner_location_abstract/models/pos_config.py

```diff
@@ -0,0 +1,25 @@
+from odoo import api, fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = "pos.config"
+
+    geolocalize_tech_name = fields.Char(compute="_compute_geolocalize")
+
+    @api.model
+    def _set_extended_data(self):
+        return {}
+
+    def _set_pos_config_parameter(self, tech_name, ext_vals=None):
+        _ = ext_vals or {}
+        for config in self:
+            config.geolocalize_tech_name = tech_name
```

Comment:
I don't understand very well why both functions are required.

Reply:
> I don't understand very well why both functions are required.

These functions helping fast and without problems configure new providers for POS.